src/NuGet.Clients/NuGet.VisualStudio/Extensibility/IVsPackageSourceProvider.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
nit: Missing BOM mark.

Reply:
Our [`.editorconfig` specifies `utf-8`](https://github.com/NuGet/NuGet.Client/blob/5b975f446808a318e0696c8c23cd2dd6b9aa817b/.editorconfig#L12) not `utf-8-bom`. Therefore any file that has a bom is going to be fighting against any editor that respects `.editorconfig`. Maybe we should consider doing a one time update of all files to remove the bom, and then any time we see bom changes we'll know the person used an editor that doesn't respect `.editorconfig`. Otherwise files can just be updated one by one when there's a PR that modifies it like in this PR, but then it's harder to detect when a bom is incorrectly added.

Reply:
Personally I'm against a one off changes. 
Been on teams/project that did that and it sucked cause it ruined all the history. We can discuss offline.